internals-js/src/operations.ts

```diff
@@ -86,7 +107,6 @@ export class Field<TArgs extends {[key: string]: any} = {[key: string]: any}> ex
     readonly alias?: string
   ) {
     super(definition.schema(), variablesInArguments(args));
-    this.validate();
```

Comment:
Note to reviewers: somewhat unrelated minor optimisation. While starting to look at this issue, I fired up profiling to see if anything specific showed up. While nothing egregious showed up, this validation came up higher than it should have, and upon reflexion, doing that validation in the constructor was a bit heavy. And since we have a `validate()` method on `SelectionSet` that we call when needed (but less aggressively), simply made it so this `Field.validate` was called only at the same time (outside of performance, this even feel a bit more expected anyway).